src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -39,6 +39,7 @@ protected function configure()
                 new InputArgument('locale', InputArgument::REQUIRED, 'The locale'),
                 new InputArgument('bundle', InputArgument::OPTIONAL, 'The bundle name or directory where to load the messages, defaults to app/Resources folder'),
                 new InputOption('prefix', null, InputOption::VALUE_OPTIONAL, 'Override the default prefix', '__'),
+                new InputOption('no-prefix', null, InputOption::VALUE_NONE, 'Should the prefix be an empty string'),
```

Comment:
Maybe we could reword this description as: `If set, no prefix is added to the translations.`